examples/stateful-goroutines/stateful-goroutines.go

```diff
@@ -1,110 +1,115 @@
-// In the previous example we used explicit locking with
-// mutexes to synchronize access to shared state across
-// multiple goroutines. Another option is to use the
-// built-in synchronization features of  goroutines and
-// channels to achieve the same result. This channel-based
-// approach aligns with Go's ideas of sharing memory by
-// communicating and having each piece of data owned
-// by exactly 1 goroutine.
+// Nell'esempio precedente abbiamo usato il lock esplicito
+// insieme alle mutex per sincronizzare l'accesso allo
+// stato condiviso da più goroutine. Un'altra opzione è
+// quella di utilizzare le funzionalità di sincronizzazione
+// native di Go, sfruttando goroutine e channel per ottenere
```

Comment:
Ah, credevo fosse singolare il nome prima per qualche motivo lol

On 18 April 2016 13:17:16 CEST, Nicola Corti <notifications@github.com> wrote:
>> @@ -1,110 +1,115 @@
>> -// In the previous example we used explicit locking with
>> -// mutexes to synchronize access to shared state across
>> -// multiple goroutines. Another option is to use the
>> -// built-in synchronization features of  goroutines and
>> -// channels to achieve the same result. This channel-based
>> -// approach aligns with Go's ideas of sharing memory by
>> -// communicating and having each piece of data owned
>> -// by exactly 1 goroutine.
>> +// Nell'esempio precedente abbiamo usato il lock esplicito
>> +// insieme alle mutex per sincronizzare l'accesso allo
>> +// stato condiviso da più goroutine. Un'altra opzione è
>> +// quella di utilizzare le funzionalità di sincronizzazione
>> +// native di Go, sfruttando goroutine e channel per ottenere
>
>Che si traduce con `native` appunto, no? :D
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/golangit/gobyexample-it/pull/27/files/ae11839da4928c62054218d153693ecd759f8201#r60041217

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.